src/main/java/domain/participant/Participant.java

```diff
@@ -0,0 +1,68 @@
+package domain.participant;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import domain.card.Card;
+
+public class Participant {
+    protected static final String SHOW_HAND_FORMAT = "%s카드: %s";
+    protected static final String JOINING_DELIMITER = ", ";
+    protected static final String SHOW_STATUS_DELIMITER = " - 결과 : ";
+    protected static final int BLACK_JACK_NUMBER = 21;
+    protected static final int ACE_COUNT_LOWER_BOUND = 0;
+    protected static final int ADDITIONAL_SCORE_ACE = 10;
+
+    protected final Name name;
+    protected List<Card> hand;
+    public final boolean isBlackJack;
```

Comment:
```suggestion
    private final Name name;
    private List<Card> hand;
    private final boolean isBlackJack;
```
필드는 private으로하시고, getter의 접근수준을 조정하는게 캡슐화에 도움될 것 같아요.

Reply:
수정했습니다🔧